string start_id

string target_id

# lookup in the time dictionary
string time_of_day

---

float64 travel_time